### DISCUSSION TITLE
Discord Rich Presence Integration

### DISCUSSION THREAD

**Discussion by CableDorado2** [ikemen-engine/Ikemen-GO#869] [Feature Request]
It would be interesting to add the necessary libraries or scripts so that when trying to link the Discord Rich Presence with Ikemen GO, it can be used and customized.

I found this: https://github.com/pfirsich/lua-discordRPC

**Comment by A983Hub** [comment]
That's a nice idea!

**Comment by Wintermourn** [comment]
I would also like support for rich presence. Those bindings can't be used though, as Ikemen GO uses GopherLua which has a modified Lua 5.1 instead of LuaJIT (which is why `require "ffi"` doesn't work).

**Comment by CableDorado2** [comment]
Probably could be done using a Go port of the library: https://github.com/hugolgst/rich-go

Original repo: https://github.com/discord/discord-rpc
